gemrb/core/Map.cpp

```diff
@@ -1343,9 +1343,9 @@ void Map::DrawSearchMap(const Region &screen)
 	block.w = 8;
 	block.h = 6;
 	while (step) {
-		block.x = (step->x+4)*16 - rgn.x;
-		block.y = (step->y+1)*12 - rgn.y - 6;
-		print("Waypoint %d at roughly (%d, %d)", i, block.x, block.y);
+		block.x = (step->x+64) - rgn.x;
+		block.y = (step->y+12) - rgn.y - 6;
+		print("Waypoint %d at (%d, %d)", i, step->x, step->y);
```

Comment:
Is this accurate now? The block coordinate calculation can also be further simplified.

Reply:
The +64/+12/-6 are still needed or it becomes less accurate. I can just make it +64/+6 though

Reply:
feel free to fix the centering math instead. :)